test/testcontroller.c

```diff
@@ -2005,6 +2005,10 @@ int main(int argc, char *argv[])
     SDL_SetHint(SDL_HINT_JOYSTICK_ALLOW_BACKGROUND_EVENTS, "1");
     SDL_SetHint(SDL_HINT_JOYSTICK_LINUX_DEADZONES, "1");
 
+#if SDL_PLATFORM_GDK
+    SDL_SetHint(SDL_HINT_JOYSTICK_GAMEINPUT, "1");
```

Comment:
We should probably enable this by default in SDL for XBox, instead of doing it here.

Reply:
Pushed a change which defaults the hint to "1" for all GDK platforms, including Windows.